include/oculusconfig.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * oculusconfig.h
+ *
+ *  Created on: Dec 29, 2020
+ *      Author: Bjorn Blissing
+ */
+
+#pragma once
+
+//#define OSGOCULUS_EXPORT __declspec( dllexport )
+//#define OSGOCULUS_EXPORT __declspec( dllimport )
+
+#ifndef OSGOCULUS_EXPORT
+#define OSGOCULUS_EXPORT
```

Comment:
I guess this should be handled by a CMake script depending on if the library is built as static och dynamic.

Reply:
I've added a stub in the latest commit which exposes an option to build the library as shared
If set, CMakes adds a preprocessor symbol used by the configuration file to define the directives. 

It's a stub only because I'd like you to decide which other settings to specify and libs to link against when building as shared lib.